lua/swenv/api.lua

```diff
@@ -6,11 +6,25 @@ local ORIGINAL_PATH = vim.fn.getenv('PATH')
 
 local current_venv = nil
 
+local update_path = function(path)
+  vim.fn.setenv('PATH', path .. '/bin' .. ':' .. ORIGINAL_PATH)
+end
+
 local set_venv = function(venv)
+  if venv.source == 'conda' then
+    vim.fn.setenv('CONDA_PREFIX', venv.path)
+    vim.fn.setenv('CONDA_DEFAULT_ENV', venv.name)
+    vim.fn.setenv('CONDA_PROMPT_MODIFIER', '(' .. venv.name .. ')')
+  else
+    vim.fn.setenv('VIRTUAL_ENV', venv.path)
+  end
+
+  vim.cmd('LspRestart')
```

Comment:
Sorry should have thought of this earlier but I think it's better to leave this up to the user. Very easy to do with the `post_set_venv` option, eg:
```lua
post_set_venv = function(venv) vim.cmd.LspRestart() end,
```

Reply:
sure, I'll add something in the readme about it

Reply:
Great! Then I think this is ready to go :)